app/views/users/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="sections">
+    <%= image_tag (user.photo), height: '115', width: '115' %>
+    <div class="users">
+        <h1><%= link_to (user.name), user%></h1>
+        <span>Number of posts: <%=user.postsCounter%></span>
+    </div>
+</div>
```

Comment:
- Great job implementing your code that displays the user.photo, user.name and user.posts_counter in this file 👍🏽